### ISSUE TITLE
Miscellaneous Bugs in the Pipeline up through Timeslide Injections

### ISSUE THREAD

**Issue by wbenoit26** [ML4GW/BBHNet#162]
1. In trainer.py, an error is raised [here](https://github.com/ML4GW/BBHNet/blob/000f666c133bfc595d8274f5bd14852feb57918e/libs/trainer/bbhnet/trainer/trainer.py#L82) because the target shape (4096) is different than the prediction shape (4096, 1). Fix this by changing [this](https://github.com/ML4GW/BBHNet/blob/000f666c133bfc595d8274f5bd14852feb57918e/projects/sandbox/train/train/validation.py#L91) line to be `y = torch.zeros((len(X), 1)))`
 
 2. The pyproject.toml in sandbox references [base.fmin](https://github.com/ML4GW/BBHNet/blob/000f666c133bfc595d8274f5bd14852feb57918e/projects/sandbox/pyproject.toml#L97). Should be `base.minimum_frequency`
 
 3. We might want to make [`clean`](https://github.com/wbenoit26/BBHNet/blob/000f666c133bfc595d8274f5bd14852feb57918e/projects/sandbox/pyproject.toml#L144) be `false` by default, just so it's easy to run this pipeline out of the box.
 
 ~~4. It looks like `max_segment_length` is no longer an argument in timeslide_injections.py, so it should be removed from the [pyproject.toml](https://github.com/wbenoit26/BBHNet/blob/000f666c133bfc595d8274f5bd14852feb57918e/projects/sandbox/pyproject.toml#L163)~~
4. `max_segment_length` in [pyproject.toml](https://github.com/wbenoit26/BBHNet/blob/000f666c133bfc595d8274f5bd14852feb57918e/projects/sandbox/pyproject.toml#L163) isn't implemented in `timeslide_injections` yet, so it should be commented out for the time being.

**Comment by alecgunny** [comment]
@wbenoit26 would you want to try your hand at fixing some of these? For (2), `hermes.quiver` now implements a `clean` kwarg to the [`ModelRepository` object](https://github.com/ML4GW/hermes/blob/d28228f77aafa05b35432da416ecd151da1fd4c0/hermes/hermes.quiver/hermes/quiver/model_repository.py#L41) itself, so you should be able to take advantage of this to make the implementation here cleaner

**Comment by wbenoit26** [comment]
Yeah, for sure, I'll give that a try.
